lib/mqtt/mqtt.cc

```diff
@@ -534,7 +541,7 @@ SObj mqtt_connect(Timeout                    *t,
 	// coreMQTT), we can assume that the allocator zeroes out for us.
 	size_t handleSize =
 	  sizeof(CHERIoTMqttContext) -
-	  sizeof(CHERIoTMqttContext::variableLengthData) + networkBufferSize +
+	  sizeof(CHERIoTMqttContext::variableLengthData) +
```

Comment:
If doing this we need to update the comment "Following this we allocate variable length data" at the end of the `CHERIoTMqttContext` object (it still mentions "the network buffer (array of uint8_t)")

Reply:
This is gone now that the RTOS doesn't impose this restriction.